onPremPackage/genericInstall.sh

```diff
@@ -248,8 +260,7 @@ function dockerLogin() {
   docker login -u "$dockerUsername" -p "$dockerPassword"
 
   if [ $? -ne 0 ]; then
-    echo "login failed, exiting."
-    exit 1
+    echo "login failed, unable to connect to docker repository."
```

Comment:
should it continue if it can't connect to the repo?

Reply:
It was the quick way of allowing the install to continue when there was no internet access and we were using a local image.

I added an execution branch that will skip the login if not needed instead and will still allow exit on failure if the login is attempted.